photon-ml/src/main/scala/com/linkedin/photon/ml/algorithm/CoordinateDescent.scala

```diff
@@ -84,34 +87,40 @@ class CoordinateDescent(
     // Initialize the training scores
     var updatedScoresContainer = coordinates.map { case (coordinateId, coordinate) =>
       val updatedScores = coordinate.score(updatedGAMEModel.getModel(coordinateId).get)
-      updatedScores.setName(s"Initialized training scores with coordinateId $coordinateId")
-          .persistRDD(StorageLevel.INFREQUENT_REUSE_RDD_STORAGE_LEVEL)
+      updatedScores
+        .setName(s"Initialized training scores with coordinateId $coordinateId")
+        .persistRDD(StorageLevel.INFREQUENT_REUSE_RDD_STORAGE_LEVEL)
```

Comment:
OK - tried to add a newline consistently throughout. We can take that up later when we iron out the photon style guide too.